transports/sqs/native-integration.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Native integration
+summary: Considerations when integrating NServiceBus endpoints with native Amazon SQS publishers and consumers.
+reviewed: 2020-12-18
+component: sqs
+versions: '[5.3,]'
+---
+
+This document describes how to consume messages from and send messages to non-NServiceBus endpoints via Amazon SQS in integration scenarios.
+
+### Access to the native Amazon SQS message details
```

Comment:
I was thinking that in theory the native message is also available in the handler and whether it would make sense to show that as well. Then I realized this is an advanced scenario and therefore a behavior as an example is better suited